packages/query/src/graphql/graphql.test.ts

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {ApolloServer, gql} from 'apollo-server-express';
+import {Pool} from 'pg';
+import {getPostGraphileBuilder} from 'postgraphile-core';
+import {plugins} from './plugins';
+
+describe('GraphqlModule', () => {
+  const pool = new Pool({
+    user: 'postgres',
+    password: 'postgres',
+    host: '127.0.0.1',
+    port: '5432',
+    database: 'postgres',
+  });
+
+  const dbSchema = 'subquery_1';
+
+  async function insertMetadata(key: string, value: string) {
+    await pool.query(`INSERT INTO subquery_1._metadata(
+            key, value, "createdAt", "updatedAt")
+            VALUES ('${key}', '${value}', '2021-11-07 07:02:31.768+00', '2021-11-07 07:02:31.768+00');`);
+  }
+
+  async function createApolloServer() {
+    const builder = await getPostGraphileBuilder(pool, [dbSchema], {
+      replaceAllPlugins: plugins,
+      subscriptions: true,
+      dynamicJson: true,
+    });
+
+    const schema = builder.buildSchema();
+
+    const server = new ApolloServer({
+      schema,
+      context: {
+        pgClient: pool,
+      },
+      cacheControl: {
+        defaultMaxAge: 5,
+      },
+      subscriptions: {
+        path: '/subscription',
+      },
+    });
+
+    return server;
+  }
+
+  beforeAll((done) => {
+    done();
+  });
+
+  beforeEach(async () => {
+    await pool.query(`CREATE SCHEMA IF NOT EXISTS ${dbSchema}`);
+    await pool.query(`CREATE TABLE IF NOT EXISTS subquery_1._metadata (
+            key character varying(255) COLLATE pg_catalog."default" NOT NULL,
+            value jsonb,
+            "createdAt" timestamp with time zone NOT NULL,
+            "updatedAt" timestamp with time zone NOT NULL,
+            CONSTRAINT _metadata_pkey PRIMARY KEY (key)
+        )`);
+  });
+
+  afterEach(async () => {
+    await pool.query(`DROP TABLE subquery_1._metadata`);
+  });
+
+  afterAll((done) => {
+    pool.close();
+    done();
+  });
+
+  it('can query all metadata fields from database', async () => {
```

Comment:
maybe another test case with some more useless records in the metadata table

Reply:
Added, although it only tests from the api. I have also checked to make sure useless records aren't being pulled out of db (after query).